change/@fluentui-react-tree-c6f8da33-6d54-4060-b38b-5ed11eb4d61e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32167&commitId=ef9e600322bf0d9e38d3f2c52f31f7fea12f0f7f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|5| Changed|

</details>


  <details><summary><b>Tree</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.size - Dark Mode.chromium.png|1263761| Changed|
|Tree.size - High Contrast.chromium.png|1246075| Changed|
|Tree.size - RTL.chromium.png|1604087| Changed|
|Tree.size.chromium.png|1604224| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>